web/src/lib/components/asset-viewer/photo-viewer.svelte

```diff
@@ -132,7 +132,11 @@
       <LoadingSpinner />
     </div>
   {:else if !imageError}
-    <div use:zoomImageAction class="h-full w-full" transition:fade={{ duration: haveFadeTransition ? 150 : 0 }}>
+    <div
+      use:zoomImageAction
+      class="h-full w-full duration-500"
```

Comment:
`duration-500` changes the duration of a transition, but there is no transition?

Reply:
Ah, missed that, sorry. It was part of something else I tried to change but reverted, and I forgot to revert that bit.

Reply:
fixed